### PR TITLE
Adm 413 BuildKite integration-Backend

### PR DESCRIPTION
**Pull request by leeranzhi**
## Summary
- update cloudformation to config AWSBuildKiteStepsRole and ssh private key.
- deploy-infra ✅
- deploy ✅

### CODE REVIEW

File: .buildkite/hooks/pre-exit [hunk 1]
```diff
@@ -12,5 +12,6 @@ echo "Start to perform cleanup"
 rm -rf /var/lib/buildkite-agent/builds/*
 rm -rf /var/lib/buildkite-agent/.cache/*
 rm -rf /var/lib/buildkite-agent/.local/*
+if docker images | grep -qE '.*<none>.*'; then docker rmi -f $(docker images -q) ; fi
```
Comment (guzhongren) [inline .buildkite/hooks/pre-exit:15]: Should delete all images before job exits agent

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -19,7 +19,18 @@ steps:
           name: "backend-license-report"
           expire_in: "${RETENTION_DAYS} days"
 
+  - label: ":rocket: Deploy to infra"
```
Comment (guzhongren) [inline .buildkite/pipeline.yml:22]: rename to `Deploy infra`

File: .buildkite/ops/deploy [hunk 3]
```diff
@@ -24,7 +28,21 @@ deploy_stub() {
 }
 
 deploy_prod() {
-  echo "deploy_prod"
+  sed -i -e "s/heartbeat_backend:latest/${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_HOST}\/heartbeat_backend:hb${BUILDKITE_BUILD_NUMBER}/g" infra/docker-compose.yml
```
Comment (guzhongren) [inline .buildkite/ops/deploy:31]: this is deploy new app,
I think, `deploy old app` in this PR or the next PR?

**Review by guzhongren — APPROVED** [review]
Good PR, just need improve some things

File: .buildkite/pipeline.yml (hunk 2 above)
Comment (leeranzhi) [reply .buildkite/pipeline.yml]: Fixed.

File: .buildkite/hooks/pre-exit (hunk 1 above)
Comment (leeranzhi) [reply .buildkite/hooks/pre-exit]: Fixed.

**Pull request merged**